src/Repository/SaferPayPaymentRepository.php

```diff
@@ -62,6 +62,16 @@ public function isLogoEnabledByName($paymentName)
         return Db::getInstance()->getValue($query);
     }
 
+    public function getAllLogosEnabled()
+    {
+        $query = new DbQuery();
+        $query->select('name');
+        $query->from('saferpay_logo');
+        $query->where('active = "1"');
+
+        return Db::getInstance()->executeS($query);
```

Comment:
executeS method has inconsistent return type.
handle the result

`    if (!$result) {
        return [];
    }
`